manpages/lumina-mediaplayer.8

```diff
@@ -0,0 +1,27 @@
+.Dd October 20, 2017
+.Dt LUMINA-MEDIAPLAYER 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-mediaplayer
+.Nd is a simple graphical utility that plays various media files. It also has hooks to stream online radio services.
+
+.Sh SYNOPSIS
+.Nm
+.Op <File Name>
+
+.Sh DESCRIPTION
+.Nm
+opens up a graphical interface that allows the user to do several things. The user can sync a Pandora account and stream music. The user can also create and manage a playlist of songs from their system. 
+.Pp
+Other than using the gui, the user is able to add a song by specifying a file name.
+
+.Sh EXAMPLES
+
+.Bl -tag -width indent
+.It lumina-mediaplayer Downloads/Testing.mp3
+Adds the song Testing.mp3 into a playlist.
```

Comment:
Is this exactly what happens when the command is run? Does it open the graphical utility? Remember to be both specific and precise about what happens. We don't want the example to do something unexpected for the reader, as that can quickly become confusing and frustrating.